core/main.cpp

```diff
@@ -156,6 +156,8 @@ static void test_exit(int test_result)
 
 int main(int argc, char *argv[])
 {
+    //std::cout<<"started main \n";
+    // double start_time = omp_get_wtime();                   //Start timer
```

Comment:
Note that the -t command-line flag to SLiM already provides a summary of total runtime.  But maybe omp_get_wtime() is providing you with different information than that provides?

Reply:
I have removed this but the reason why I use omp_get_wtime() is that it tracks time better for multithreaded segments.